source/Variant.h

```diff
@@ -0,0 +1,63 @@
+/* Variant.h
+Copyright (c) 2019 by Michael Zahniser
```

Comment:
I would expect you to have created this file.

Reply:
Just because I created the file doesn't mean the copyright needs to be to me.

Reply:
Okay, good to hear this is not a copy-paste issue but a deliberate decision.